src/client/components/auth/LoginForm.component.tsx

```diff
@@ -92,7 +91,7 @@ const LoginForm: FunctionComponent = () => {
     <div className={styles.cardWrapper}>
       <div className={styles.cardContainer}>
         <Link href="/">
-          <FontAwesomeIcon icon={faTimes} className={styles.cancelIcon} />
+          <XLg className={styles.cancelIcon} color={"grey"} size={40} />
```

Comment:
Tu peux enlever les accolades en trop
```suggestion
          <XLg className={styles.cancelIcon} color="grey" size={40} />
```